mungegithub/OWNERS

```diff
@@ -8,3 +8,5 @@ reviewers:
 - eparis
```

Comment:
Cannot parse file: error converting YAML to JSON: yaml: line 13: could not find expected ':'.